lib/src/chart/base/base_chart/base_chart_data.dart

```diff
@@ -17,21 +17,26 @@ abstract class BaseChartData with EquatableMixin {
   BaseChartData({
     FlBorderData? borderData,
     required this.touchData,
-  }) : borderData = borderData ?? FlBorderData();
+    BorderRadius? borderRadius,
```

Comment:
We don't need to have `borderRadius` property here in our base chart data. (So we don't need to have it for all the charts. And if we do, we should have it around the chart. Not something that only applies for the pie section paths)
So please remove it from here and just add it inside the `PieChartSectionData`